src/integration/pages/homePage/canonicalTests.js

```diff
@@ -14,14 +14,29 @@ export default ({ service, pageData, displayAds }) => {
     runCanonicalAdsTests();
   }
 
-  it('should render the correct number of curations, including most read', () => {
-    const curationsWithSummaries = pageData.curations.filter(
-      ({ summaries, mostRead }) =>
-        (summaries && summaries?.length > 0) || mostRead,
-    );
+  describe('Most Read', () => {
+    it('should render the correct number of curations, including most read', () => {
```

Comment:
And we can add the radio schedule to this test description 
```suggestion
    it('should render the correct number of curations, including most read & radio schedule', () => {
```